pandas/io/formats/style.py

```diff
@@ -1480,3 +1518,13 @@ def _maybe_wrap_formatter(formatter):
             "instead".format(formatter=formatter)
         )
         raise TypeError(msg)
+
+
+def _maybe_wrap_na_formatter(formatter, na_rep):
```

Comment:
Can this be folded into `_maybe_wrap_formatter`? I'm finding the multiple layers of wrapping hard to follow.

Reply:
No problem, all done!